src/constants.js

```diff
@@ -0,0 +1,9 @@
+export const ALLOWED_EXTENSIONS = [
+	'html',
+	'htm',
+	'pdf',
+];
+
+// Configured constants
```

Comment:
I mean, this is crazy pedantic, but aren't these all "configured constants"? These are _environment_-configured constants. 😛

Reply:
What is a configuration but a variable pancake

Reply:
:-|

Reply:
🥞